scripts/check_frontend_test_coverage.py

```diff
@@ -212,6 +211,7 @@
     'subtopic-summary-tile.component.ts',
     'subtopic.model.ts',
     'suggestion-modal-for-exploration-editor.service.ts',
+    'summarize-nonnegative-number.pipe.ts',
```

Comment:
Hmm, @ashutoshc8101 why did you suggest to add this? Can we instead cover it with tests?

Reply:
@nithusha21 This is a valid error in develop. `summarize-nonnegative-number.pipe.ts` file is not covered and is neither added to exclude list. But somehow, frontend coverage test still passes. Weird!.
This PR doesnot make changes to these files but cause these valid errors to show up.